addons/parachute/XEH_preInit.sqf

```diff
@@ -21,4 +21,12 @@ PREP_RECOMPILE_START;
 #include "XEH_PREP.hpp"
 PREP_RECOMPILE_END;
 
+[
+    QGVAR(hideAltimeter),
+    "CHECKBOX",
+    [LSTRING(HideAltimeter), LSTRING(HideAltimeter_tooltip)],
+    format ["ACE %1", localize ELSTRING(common,DisplayName)],
+    true
```

Comment:
We might want to add
```
    true,
    {[QGVAR(hideAltimeter), _this, false] call EFUNC(common,cbaSettings_settingChanged)}
```
to make it global and then to add a note that a restart might be required for this setting change to take effect.

In testing the soldier UI didn't always reload after jumping, so once those UI are hidden they won't come back if setting is changed back to easy.

But this would be very rare issue.

Reply:
Why make it a global setting needlessly? That would just be a restriction and this one works perfectly fine as individual setting.
Good point about the warning though.